src/test/kotlin/io/github/gradlenexus/publishplugin/TaskOrchestrationTest.kt

```diff
@@ -106,6 +106,32 @@ class TaskOrchestrationTest {
         assertThat(simplifiedCloseAndReleaseTasks.first().dependsOn).isEmpty()
     }
 
+    @Test
+    internal fun `simplified close task without repository name should be available but trigger nothing if no repositories are configured`() {
+        initSingleProjectWithDefaultConfiguration()
+        project.extensions.configure<NexusPublishExtension> {
+            repositories.clear()
+        }
+
+        val simplifiedCloseTasks = project.getTasksByName(NexusPublishPlugin.SIMPLIFIED_CLOSE_TASK_NAME, true)
+
+        assertThat(simplifiedCloseTasks).hasSize(1)
+        assertThat(simplifiedCloseTasks.first().dependsOn).isEmpty()
+    }
+
+    @Test
+    internal fun `simplified release task without repository name should be available but trigger nothing if no repositories are configured`() {
+        initSingleProjectWithDefaultConfiguration()
+        project.extensions.configure<NexusPublishExtension> {
+            repositories.clear()
+        }
+
+        val simplifiedReleaseTasks = project.getTasksByName(NexusPublishPlugin.SIMPLIFIED_RELEASE_TASK_NAME, true)
+
+        assertThat(simplifiedReleaseTasks).hasSize(1)
+        assertThat(simplifiedReleaseTasks.first().dependsOn).isEmpty()
+    }
```

Comment:
What do you think about using the parameterized tests for those two sibling cases?

Reply:
Sure, can you point me to an example maybe? I'm not too much into Jupiter, I'm still largely preferring Spock. :-)

Reply:
:-D

https://github.com/gradle-nexus/publish-plugin/blob/00d5611cfdee82ecd2e26eeef1aac40e61a9a246/src/e2eTest/kotlin/io/github/gradlenexus/publishplugin/e2e/NexusPublishE2ETests.kt#L29-L31